components/Card/CardTitle.tsx

```diff
@@ -27,93 +13,151 @@ interface CardTitleProps {
-      </div>
-      <CardBootstrap.Body className="px-3 py-0">
+  switch (type) {
+    case "testimony":
+      return (
+        <>
+          {header && subheader && (
+            <CardBootstrap.Title
+              className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+            >
+              {subheader} {t("newsfeed.endorsed")}
+              <a href={`/bills/${court}/${header}`}>
+                <strong>{formatBillId(header)}</strong>
+              </a>
+            </CardBootstrap.Title>
+          )}
+        </>
+      )
+    case "bill":
+      return (
+        <>
+          {header && (
+            <CardBootstrap.Title
+              className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+            >
+              <a href={`/bills/${court}/${header}`}>
+                <strong>{formatBillId(header)}</strong>
+              </a>{" "}
+              {subheader && (
+                <>
+                  {t("newsfeed.action_update")}
+                  {subheader}
+                </>
+              )}
+            </CardBootstrap.Title>
+          )}
+        </>
+      )
+    default:
+      return (
+        <CardBootstrap.Title
+          className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+        >
+          <strong>{header}</strong>
+        </CardBootstrap.Title>
+      )
+  }
+}
+
+const CardTitleFollowing = (props: CardTitleProps) => {
+  const { header, subheader, isBillMatch, isUserMatch, type } = props
+  const { t } = useTranslation("common")
+
+  if (type == `no results`) {
+    return <></>
+  } else if (type === `bill`) {
+    return (
+      <>
         {header && (
-          <CardBootstrap.Title
-            className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
-          >
-            <a href={`/bills/${court}/${header}`}>
-              <strong>{formatBillId(header)}</strong>
-            </a>{" "}
-            {subheader ? (
-              <>
-                {t("newsfeed.action_update")}
-                {subheader}
-              </>
-            ) : (
-              <></>
-            )}
-          </CardBootstrap.Title>
-        )}
-        {header ? (
           <CardBootstrap.Title
             className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
           >
-            {header && isBillMatch ? (
+            {isBillMatch ? (
               <>{t("newsfeed.follow")}</>
             ) : (
               <>{t("newsfeed.not_follow")}</>
             )}
             <strong>{formatBillId(header)}</strong>
           </CardBootstrap.Title>
-        ) : (
-          <></>
         )}
-      </CardBootstrap.Body>
-    </CardBootstrap.Body>
-  )
+      </>
+    )
+  } else if (isBillMatch === true && isUserMatch === true) {
+    return (
+      <CardBootstrap.Title
+        className={`align-items-start fs-6 lh-sm mb-1 text-body-tertiary`}
+      >
+        {t("newsfeed.follow")}
+        {header && <strong>{formatBillId(header)}</strong>}
```

Comment:
I know this isn't an issue with this PR, but I wanted to bring it up - I don't love that `header` and `subheader` have implicit meanings beyond being what's rendered into the notification. If we need the extra formatting around billId/links, we should just supply the billId + notification type as props and handle the formatting with those.